### ISSUE TITLE
Chest ammo pouch doesn't display explicit pocket restriction GRENADE

### ISSUE THREAD

**Issue by utechenko-v** [CleverRaven/Cataclysm-DDA#60971]
### Describe the bug

Chest ammo pouch doesn't display explicitly in pocket info that the pocket accepts only grenades.
Other pouches display explicit pocket restriction, for example, chest ammo pouch has "* Item must be a compact magazine".

### Steps to reproduce

1. Find or debug spawn a chest ammo pouch
2. View item, check pocket restrictions
Actual result: just "Restrictions: " without actual restriction displayed

### Expected behavior

Pocket info should show that it has restrictions - accepts only grenades. (Flag Restriction GRENADE)
e.g.: "Restrictions:  * Item must be a grenade"

### Screenshots

![image](https://user-images.githubusercontent.com/16246933/190011532-6b00ce52-e044-4957-8973-6aeb3a7213cc.png)

### Versions and configuration

build type: windows-tiles-x64
build number: 2022-09-13-0704
commit sha: 9c889520b13268ce8a431a1c9e054b6ee9ae6ca3

**Comment by epsimpson** [comment]
Yes, this was driving me crazy.  I found a US Ballistic Vest with various pouches attached.  Wanted to put medical supplies in the apparently unlimited "grenade pouch".  When I insert bandages(33) into the vest, it always put bandages(7) in a small (0.25 L) HOLSTER "gas mask pouch", even after I blacklisted it.

Interestingly, when I try to insert the remaining bandages(26) in the vest again ( hoping they'll go into the grenade pouch, and the vest lets me try), it throws the error:

 DEBUG    : tried to put an item (bandages, amount 0) in a container (ballistic_vest_esapi) that cannot contain it

 FUNCTION : int item::fill_with(const item&, int, bool, bool, bool)
 FILE     : src/item.cpp
 LINE     : 12090
 VERSION  : 0.F-11212-g8f4403757c

I'm curious why all these pouches have such specific limitations in the first place.  Generally a container is a container, but perhaps in this case the grenades clip on so not an actual pouch?

**Comment by catdach** [comment]
Looks like the problem is that the "grenade" flag doesn't have a defined "restriction" message in 
`CleverRaven/Cataclysm-DDA//data/json/flags.json`
should be an easy fix. I'll get a PR for it soon.
